### ISSUE TITLE
I can't understand the pooling ops on the remote frames, please tell me how to compute it?

### ISSUE THREAD

**Issue by Leiyi-HU** [GuoleiSun/VSS-CFFM#8]
`            for k in range(len(focal_l_clips)):
                # window_size_glo = math.floor(self.window_size_glo / (2 ** k))
                if focal_l_clips[k]>self.window_size:
                    window_size_glo = focal_l_clips[k]
                else:
                    window_size_glo = math.floor(self.window_size_glo / (focal_l_clips[k])) # 7 / [7,4,2] -> 1,1,3
                # window_size_glo = focal_l_clips[k]
                if self.pool_method == "fc":
                    self.pool_layers_clips.append(nn.Linear(window_size_glo * window_size_glo, 1))
                    self.pool_layers_clips[-1].weight.data.fill_(1./(window_size_glo * window_size_glo))
                    self.pool_layers_clips[-1].bias.data.fill_(0)
                elif self.pool_method == "conv":
                    self.pool_layers_clips.append(nn.Conv2d(dim, dim, kernel_size=window_size_glo, stride=window_size_glo, groups=dim))`
In the code above, I think it should be `focal_l_clips[k]>self.window_size`, for example, the pooling window or kernel should be 7 at the 0th feature, and 4 at the index 1 features, and 2 at the index 2 features. Maybe I misunderstood it,  so could you please tell the logic behind the code above?

**Comment by GuoleiSun** [comment]
Thanks for your interest. As you noticed, focal_l_clips controls the pooling window for previous frames before the target one. It is a parameter which could be set. In our code, we set it to be [1,2,3]. We have modified it [(here)](https://github.com/GuoleiSun/VSS-CFFM/blob/8537cb54769fbdd5e67c75d4835bb3d90d0318e9/mmseg/models/decode_heads/cffm_head.py#L105) and updated our trained weights [(here)](https://drive.google.com/drive/folders/1YD5Yy6_m3QlS72o6FQWmsFtz7Kw-8OmI?usp=sharing)

**Comment by Leiyi-HU** [comment]
> Thanks for your interest. As you noticed, focal_l_clips controls the pooling window for previous frames before the target one. It is a parameter which could be set. In our code, we set it to be [1,2,3]. We have modified it [(here)](https://github.com/GuoleiSun/VSS-CFFM/blob/8537cb54769fbdd5e67c75d4835bb3d90d0318e9/mmseg/models/decode_heads/cffm_head.py#L105) and updated our trained weights [(here)](https://drive.google.com/drive/folders/1YD5Yy6_m3QlS72o6FQWmsFtz7Kw-8OmI?usp=sharing)

Thank you for your reply, that helps me a lot! By the way, I have read another paper of you(eccv), which is also a wonderful work like this one!

**Comment by GuoleiSun** [comment]
Thanks very much
